Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-data-processing.view.html

```diff
@@ -0,0 +1,21 @@
+﻿<div class="object-section-container" ng-if="isDataProcessingEnabled">
+    <h4>{{systemUsageName}}</h4>
+    <div class="panel panel-default" >
+        <div class="panel-heading"><strong>Databehandling</strong></div>
+
+        <table class="table table-striped table-condensed table-fixed" ng-show="filterDataProcessor" data-element-type="dataProcessingRegistrationView">
```

Comment:
please remove the class `table-condensed`

Reply:
don't use `data-element-type` .. it's only meant to be used where we cannot set a proper `id` (like on kendo elements)

Remember to fix in the tests if you use it there

Reply:
use ng-if in stead of show and bind to the length of the collection in stead. no need for an additional property here